<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QWidget" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>684</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>684</width>
    <height>720</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pictures.qrc">
    <normaloff>:/mainLogo/resources/5.png</normaloff>:/mainLogo/resources/5.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#RegistrationWindow{ background-color: rgb(38, 50, 76);
color: rgb(0, 170, 127);}</string>
  </property>
  <widget class="QPushButton" name="pushButtonGoToMainWindow">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 127);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
   </property>
   <property name="text">
    <string>Create account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGoToLoginWindow">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>660</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 127);
background-color: rgb(67, 81, 108);
font: 10pt &quot;Britannic Bold&quot;;</string>
   </property>
   <property name="text">
    <string>Sing in</string>
   </property>
  </widget>
  <widget class="QLabel" name="Logo">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>30</y>
     <width>741</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pictures.qrc">:/mainLogo/resources/10.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>481</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="labelSurname">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>*Surname</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditSurname">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
border: rgb(84, 96, 117);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditName">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
border: rgb(84, 96, 117);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelName">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>*Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="labelPassportSeries">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>*Passport series</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelPhoneNumber">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>*Phone number</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditPhoneNumber">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
border: rgb(84, 96, 117);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;
</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPassportValidity">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>*Passport validity</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPassportValidity">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
border: rgb(84, 96, 117);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditPassportSeries">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
border: rgb(84, 96, 117);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelCitizenship">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>*Citizenship</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditCitizenship">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
border: rgb(84, 96, 117);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="labelEmail">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>*E-mail</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPassword">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
border: rgb(84, 96, 117);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditLogin">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
border: rgb(84, 96, 117);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditEmail">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>67</red>
              <green>81</green>
              <blue>108</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>67</red>
              <green>81</green>
              <blue>108</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>67</red>
              <green>81</green>
              <blue>108</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="NoBrush">
             <color alpha="128">
              <red>0</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>67</red>
              <green>81</green>
              <blue>108</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>67</red>
              <green>81</green>
              <blue>108</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>67</red>
              <green>81</green>
              <blue>108</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="NoBrush">
             <color alpha="128">
              <red>0</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>67</red>
              <green>81</green>
              <blue>108</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>67</red>
              <green>81</green>
              <blue>108</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>67</red>
              <green>81</green>
              <blue>108</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="NoBrush">
             <color alpha="128">
              <red>0</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
border: rgb(84, 96, 117);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPassword">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>*Password</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelLogin">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>*Login</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="pictures.qrc"/>
 </resources>
 <connections/>
</ui>
